tests/modules.py

```diff
@@ -108,6 +133,49 @@ def test_module_import(network, args):
     return network
 
 
+@reqs.description("Test Node.js/npm app")
+def test_npm_app(network, args):
+    primary, _ = network.find_nodes()
+
+    LOG.info("Building npm app")
+    app_dir = os.path.join(THIS_DIR, "npm-app")
```

Comment:
Could we do this under `build` (or whatever the current directory is when the test is run), rather than in `tests` with additional `.gitignore`? As far as I can see, that should be fine for the `modules` test, and only makes the demo HTML page a little more awkward?

Reply:
npm doesn't support out-of-tree installs, so we would have to copy the project to the build folder first. Should I do that?

Reply:
After further discussion, we're happy to leave this under `tests/` for now, as this is how npm packages are usually built.